client/src/pages/Login.js

```diff
@@ -24,28 +28,49 @@ class LoginPage extends Component {
 
   validate = () => {
     let emailError = "";
-    var re = /[^@]+@[^.]+\..+/;
+    var re = /[^@]+@[^.]+..+/;
     var test = re.test(this.state.email);
     if (!test === true) {
       emailError = "Invalid email";
     }
-
+    
     if (emailError) {
       this.setState({ emailError });
       return false;
     }
 
+    this.setState({ emailError: undefined });
     return true;
   };
 
   handleSubmit = event => {
     event.preventDefault();
     const isValid = this.validate();
+    
     if (isValid) {
-      console.log(this.state);
-      this.setState(initalState);
+      const { email, password } = this.state;
+
+      const data = {
+        email: email,
+        password: password
+      }
+
+      axios.post('/users/login', data)
+      .then(res => {
+          const { token } = res.data;
+          const decoded = jwt_decode(token);
+          localStorage.setItem('jwtToken', res.data);
```

Comment:
I missed this, but you should put the token in the localStorage, not res.data

Reply:
You are right! I'll fix it